### DISCUSSION TITLE
What t fk>?

### DISCUSSION THREAD

**Discussion by vizvasrj** [LunarWatcher/lunarwatcher.github.io-old#45] [General]
![image](https://user-images.githubusercontent.com/83173578/176862601-a8997487-74f1-47bb-9615-8208aefd66b9.png)
What is this??

**Comment by LunarWatcher** [comment]
A troll.

The situation has been dealt with. Reported to GitHub yesterday, who responded in about 32 minutes (which is amazingly fast; other sites have several days of waiting time regardless of urgency). Also reported in a few other strategic and applicable places. The user has been banned from GitHub, so all good for now. 

It was targeted at me. You presumably got emailed about it because you watch the repo? Unfortunately, getting emails whenever trolls decide to attack is a risk with watching any repo; regardless of what the repo is for, or who it's owned by.

 I do hope nothing like this happens again though, but it's unfortunately not possible to guarantee, because this is the internet and some people just want to spread hate. 

I take care of stuff like this as soon as it shows up at least (provided I'm awake), but that's damage control and not prevention (which I can't do; only GitHub can do anything on that front).

**Reply by vizvasrj** [reply]
Sorry for disturbance.

**Reply by LunarWatcher** [reply]
Don't be! I'd be pretty curious too. They also attacked some existing comment sections, so I imagine other people who don't watch the repo itself got the email as well. Keeping a public record for them is probably a good idea.

**Reply by vizvasrj** [reply]
you must be brave.
